lib/checkunusedfunctions.cpp

```diff
@@ -240,6 +240,10 @@ void CheckUnusedFunctions::parseTokens(const Tokenizer &tokenizer, const Setting
             funcname = tok->next();
             while (Token::Match(funcname, "%name% :: %name%"))
                 funcname = funcname->tokAt(2);
+        } else if (Token::Match(tok, "& %name%") && tok->scope()->isLocal()) {
+            funcname = tok->next();
+            while (Token::Match(funcname, "%name% :: %name%"))
```

Comment:
If you really want to handle namespaces, ":: %name% :: ..." should also be considered.